R/utils-shared.R

```diff
@@ -219,5 +219,5 @@ isEmptyOrBlank <- function(val = NULL, listObjects = NULL, objectName = NULL){
 isEmptyVar <- function(variable){
   all(!is.null(variable), 
       nrow(variable) != 0 || is.null(nrow(variable)), 
-      length(variable$time[!is.na(variable$time)]) != 0 & length(variable$value[!is.na(variable$value)]) != 0)
+      length(variable$time[!is.na(variable$time)]) != 0 & length(variable$value) != 0)
```

Comment:
I'm wondering if we need to check `time` and `value`? If there are non-NA times, then we would consider that variable not empty. Not really sure we need to check for length of variable anyways. Are there really cases where there are non-NA times, but there are no values listed? That would seem like bad data